composer.json

```
@@ -36,10 +40,19 @@
     },
     "scripts" :{
         "test": ["@cs", "phpunit"],
-        "tests": ["rm -rf tests/tmp/", "@cs", "phpstan analyse -c phpstan.neon --no-progress", "psalm --show-info=false", "@test"],
+        "tests": ["rm -rf tests/tmp/", "@cs", "@sa", "@test"],
         "coverage": ["php -dzend_extension=xdebug.so ./vendor/bin/phpunit --coverage-text --coverage-html=build/coverage"],
-        "cs": ["php-cs-fixer fix -v --dry-run", "phpcs --standard=./phpcs.xml src"],
-        "cs-fix": ["php-cs-fixer fix -v", "phpcbf src"],
-        "metrics": ["phpmetrics --report-html=build/metrics/ --extensions=php src,composer.json,composer.lock"]
+        "cs": ["phpcs --standard=./phpcs.xml src tests"],
```

phpcs も `reqire-dev` に追加してvendor配下を参照するのが良いと思います。

```suggestion
        "cs": ["./vendor/bin/phpcs --standard=./phpcs.xml src tests"],
```

---

そうですね！